validations/src/tests.rs

```diff
@@ -1801,7 +1893,10 @@ fn data_request_value_overflow() {
     });
     assert_eq!(
         x.unwrap_err().downcast::<TransactionError>().unwrap(),
-        TransactionError::FeeOverflow,
+        TransactionError::DataRequestWeightLimitExceeded {
+            weight: 41680557,
+            max_weight: 80000
+        },
```

Comment:
I think you can avoid this change by increasing the max dr weight used inside `test_drtx`